Jenkinsfile

```diff
@@ -31,12 +31,12 @@ node('ubuntu-zion') {
       deleteDir()
       OsTools.runSafe(this, "docker system prune -a -f")
 
-      checkout scm
+      def checkoutDetails = checkout scm
 
       dockerFileLocation = "${pwd()}/Dockerfile"
 
-      branch = OsTools.runSafe(this, 'git rev-parse --abbrev-ref HEAD')
-      commitId = OsTools.runSafe(this, 'git rev-parse HEAD')
+      branch = checkoutDetails.GIT_BRANCH
```

Comment:
branch now equals 'origin/master' which is good. The master branch check is checking for 'master'.

Reply:
I like the checkoutDetails object 👍

Reply:
Normalized it like [0]. AFAICT there's no way to always get the local branch name and this is required to checkout attached to update version and sha.

[0] https://github.com/sonatype/docker-nexus3/pull/94/commits/633de316c938321517faa649e8a68cf1355f1d5c#diff-58231b16fdee45a03a4ee3cf94a9f2c3R38